FerreteriaMaresa/Datos/Procedimiento.sql

```diff
@@ -0,0 +1,204 @@
+use Ferreteria_Maresa
+
+-------------------------PROCEDIMIENTOS ALMACENADOS-------------------------
+----EMPLEADO----
+
+create procedure validarUsuario
+@userName nvarchar(13),
+@psw varchar(80)
+as begin
+if exists(select id_empleado, contrasenia from Empleados where id_empleado = @userName and contrasenia = @psw)
+          select * from Empleados;
```

Comment:
El if no funcionara porque contrasenia es parte de la tabla usuario, hacer joins

no es select*from empleados, para validar usuario se haria con joins
ejemplo
select * empleados from inner join
usuario on @usuario=usuario and contrasenia=@contrasenia and usuario=empleado